python/ray/tune/logger.py

```diff
@@ -221,12 +221,15 @@ def close(self):
 
 
 def to_tf_values(result, path):
+    from tensorboardX.summary import make_histogram
```

Comment:
@richardliaw we are using TBXLogger by default now right? So in Ray latest to_tf_values won't be called.

I think this logic should be moved to the TBXLogger class.

Reply:
(we should also delete this old code).

Reply:
@roireshef can you push this logic into the TBXLogger class?

Reply:
I think I got it right, but can't test because I can't get this version to run inside my docker container yet (see https://github.com/ray-project/ray/issues/6662). It might take me a while to figure out, so I'd be glad to get your help testing it if you could...